tunnel/intra/udp.go

```diff
@@ -27,16 +27,29 @@ import (
 	"github.com/eycorsican/go-tun2socks/core"
 )
 
+// Summary of a non-DNS UDP association, reported when it is discarded.
+type UDPSocketSummary struct {
+	Upload   int64 // Amount uploaded (bytes)
```

Comment:
Consider appending the unit to the name (i.e. UploadBytes)

Reply:
Done